skore/tests/unit/sklearn/plot/roc_curve/test_comparison_cross_validation.py

```diff
@@ -232,3 +232,79 @@ def test_multiclass_classification_kwargs(pyplot, multiclass_classification_repo
     display.plot(despine=False)
     assert display.ax_[0].spines["top"].get_visible()
     assert display.ax_[0].spines["right"].get_visible()
+
+
+def test_data_source_binary_classification(pyplot, binary_classification_data_no_split):
```

Comment:
Can you check that the results when changing the data_source do change please?  
For instance, you can use a subsample of X and y when giving the data source `X_y` so you don't have to create a new one, and then check that the dataframes `display.frame()` outputed are different with `assert not` and the [equals](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.equals.html) function.